### PR TITLE
[MAINTENANCE] Rearrange integration tests to insure categorization into proper deployment-style based lists

### PR DESCRIPTION
**Pull request by alexsherstinsky**
### Scope
This is follow up from `https://github.com/great-expectations/great_expectations/pull/5430#discussion_r912285296` (in #5430) and conversation with @Shinnnyshinshin about how deployment-style specific lists are composed.

#### Note
Please review only `test_script_runner.py` (the other changes will be auto-merged from #5470).  Thank you. 

Please annotate your PR title to describe what the PR does, then give a brief bulleted description of your PR below. PR titles should begin with [BUGFIX], [FEATURE],  [DOCS], or [MAINTENANCE]. If a new feature introduces breaking changes for the Great Expectations API or configuration files, please also add [BREAKING]. You can read about the tags in our [contributor checklist](https://docs.greatexpectations.io/docs/contributing/contributing_checklist).

Changes proposed in this pull request:
- GREAT-727/GREAT-902
-
-

After submitting your PR, CI checks will run and @cla-bot will check for your CLA signature.

For a PR with nontrivial changes, we review with both design-centric and code-centric lenses.

In a design review, we aim to ensure that the PR is consistent with our relationship to the open source community, with our software architecture and abstractions, and with our users' needs and expectations. That review often starts well before a PR, for example in github issues or slack, so please link to relevant conversations in notes below to help reviewers understand and approve your PR more quickly (e.g. `closes #123`).

Previous Design Review notes:
-
-
-

### Definition of Done
Please delete options that are not relevant.

- [x] My code follows the Great Expectations [style guide](https://docs.greatexpectations.io/docs/contributing/style_guides/code_style)
- [x] I have performed a [self-review](https://docs.greatexpectations.io/docs/contributing/contributing_checklist) of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] I have added [unit tests](https://docs.greatexpectations.io/docs/contributing/contributing_test#writing-unit-and-integration-tests) where applicable and made sure that new and existing tests are passing.
- [x] I have run any local integration tests and made sure that nothing is broken.

Thank you for submitting!

### CODE REVIEW

File: tests/render/test_inline_renderer.py [hunk 1]
```diff
@@ -32,14 +32,6 @@
                         "schema": {"type": "com.superconductive.rendered.string"},
                         "params": {
                             "value": {"schema": {"type": "number"}, "value": 3},
-                            "row_condition": {
```
Comment (Shinnnyshinshin) [inline tests/render/test_inline_renderer.py]: :heart: thanks for fixing this

File: tests/test_utils.py [hunk 2]
```diff
@@ -566,6 +567,21 @@ def load_and_concatenate_csvs(
     return all_dfs_concatenated
 
 
+def convert_string_columns_to_datetime(
+    df: pd.DataFrame, column_names_to_convert: Optional[List[str]] = None
+) -> None:
+    """
+    Converts specified columns (e.g., "pickup_datetime" and "dropoff_datetime") to datetime column type.
+    Side-effect: Passed DataFrame is modified (in-place).
```
Comment (Shinnnyshinshin) [inline tests/test_utils.py]: Very helpful mention in the Docstring :) Thank you

**Review by anthonyburdi — APPROVED** [review]
LGTM!

File: great_expectations/execution_engine/split_and_sample/sqlalchemy_data_splitter.py [hunk 3]
```diff
@@ -196,20 +196,24 @@ def split_on_column_value(column_name: str, batch_identifiers: dict) -> bool:
...
         batch_identifiers: dict,
         date_format_string: str = "%Y-%m-%d",
     ) -> bool:
         """Convert the values in the named column to the given date_format, and split on that"""
-
-        return (
-            sa.func.strftime(
-                date_format_string,
-                sa.column(column_name),
+        if self._dialect == "sqlite":
```
Comment (Shinnnyshinshin) [inline great_expectations/execution_engine/split_and_sample/sqlalchemy_data_splitter.py]: :heart:

**Review by Shinnnyshinshin — APPROVED** [review]
Thank you for this Alex. LGTM

**Pull request merged**